internal/main.go

```diff
@@ -85,6 +85,7 @@ func main() {
 		FetchFunc:         oemConfig.FetchFunc(),
 		OemBaseConfig:     oemConfig.BaseConfig(),
 		DefaultUserConfig: oemConfig.DefaultUserConfig(),
+		OEM:               string(flags.oem),
```

Comment:
Good point. I just changed this to pass the `oemConfig` directly into the engine, where the lookups are now performed directly. I think this looks a good bit cleaner.